frontend-react/src/components/MessageTracker/MessageDetails.tsx

```diff
@@ -15,57 +18,157 @@ type MessageDetailsProps = {
     id: string | undefined;
 };
 
+const parseFileLocation = (
+    fileLocation: string
+): {
+    fileLocation: string;
+    sendingOrg: string;
+    fileName: string;
+} => {
+    let fileLocationFromStr = "";
+    let sendingOrgFromStr = "";
+    let fileNameFromStr = "";
+
+    let fileReportsLocation = fileLocation.split("reports/").pop();
+    if (fileLocation.includes("reports/") && fileReportsLocation) {
+        fileLocationFromStr = fileReportsLocation.split("%2F")[0].toUpperCase();
+        sendingOrgFromStr = fileReportsLocation.split("%2F")[1];
+        fileNameFromStr = fileReportsLocation.split("%2F")[2];
+    }
+
+    return {
+        fileLocation: fileLocationFromStr,
+        sendingOrg: sendingOrgFromStr,
+        fileName: fileNameFromStr,
+    };
+};
+
+const dataToAccordionItems = (props: {
+    id: string;
+    title: string;
+    data: WarningError[];
+}): AccordionItemProps[] => {
+    const results: AccordionItemProps[] = [];
+    if (props.data.length === 0) {
+        return [];
+    }
+    const items = (
+        <WarningsErrors title={""} data={props.data.map((md) => md.detail)} />
+    );
+    results.push({
+        id: props.id,
+        title: props.title,
+        expanded: false,
+        content: items,
+        headingLevel: "h3",
+        className: "",
+    });
+
+    return results;
+};
+
 export function MessageDetails() {
     const navigate = useNavigate();
     const { id } = useParams<MessageDetailsProps>();
     const { messageDetails } = useMessageDetails(id!!);
+    const warnings: WarningError[] = messageDetails
+        ? messageDetails.warnings
+        : [];
+    const errors: WarningError[] = messageDetails ? messageDetails.errors : [];
+
+    const fileUrl = messageDetails?.fileUrl ? messageDetails.fileUrl : "";
+    const { fileLocation, sendingOrg, fileName } = parseFileLocation(fileUrl);
 
     return (
         <>
             <div
                 className="grid-container margin-bottom-10"
                 data-testid="container"
             >
-                <div className="margin-bottom-5">
-                    <h1>Message Details</h1>
-                    <Button onClick={() => navigate(-1)} type="button">
+                <div>
+                    <h2>Message ID Search</h2>
+                    <Button
+                        onClick={() => navigate("/admin/message-tracker")}
+                        type="button"
+                    >
                         Back
                     </Button>
                 </div>
-                <DetailItem
-                    item={"Message ID"}
-                    content={messageDetails!.messageId}
-                ></DetailItem>
+                <hr className="margin-top-4 margin-bottom-4" />
+                <div className="display-flex flex-column">
+                    <span className="text-base font-sans-sm">Message ID</span>
+                    <h1 className="margin-top-1">
```

Comment:
Let's try to use a single direction for vertical margins: either top or bottom, but not both.  [Collapsing margins](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Box_Model/Mastering_margin_collapsing) can be a pain to work around, especially when we're trying to establish a predictable vertical rhythm between elements on the page.